src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
```

Comment:
There is a compilation error. Kindly resolve it and make sure the app is working without any error
![errors](https://user-images.githubusercontent.com/35504149/170129124-068fae59-0a29-4eda-b850-2880b3a21191.png)
.

Reply:
It does not appear on my part, when you run npm start everything works perfect

Reply:
Please check it again it is working properly I tried it many times